src/components/radar/__tests__/RadarAnimation.test.tsx

```diff
@@ -13,81 +13,81 @@ describe('components/radar/RadarAnimation', () => {
         },
         {
             // description:
             //     'should render correctly with middle-center positioning',
-            position: 'middle-center',
+            position: RadarAnimationPosition.MIDDLE_CENTER,
         },
         {
             // description: 'should render correctly with middle-left positioning',
-            position: 'middle-left',
+            position: RadarAnimationPosition.MIDDLE_LEFT,
         },
         {
             // description:
             //     'should render correctly with middle-right positioning',
-            position: 'middle-right',
+            position: RadarAnimationPosition.MIDDLE_RIGHT,
         },
         {
             // description: 'should render correctly with top-center positioning',
-            position: 'top-center',
+            position: RadarAnimationPosition.TOP_CENTER,
         },
         {
             // description: 'should render correctly with top-left positioning',
-            position: 'top-left',
+            position: RadarAnimationPosition.TOP_LEFT,
         },
         {
             // description: 'should render correctly with top-right positioning',
-            position: 'top-right',
+            position: RadarAnimationPosition.TOP_RIGHT,
         },
     ].forEach(({ position }) => {
         test(`should render correctly with ${position} positioning`, () => {
-            const wrapper = getWrapper({ position });
+            const wrapper = getWrapper({ position } as RadarAnimationProps);
             expect(wrapper).toMatchSnapshot();
         });
     });
 
     describe('isShown', () => {
         test('should be shown when isShown is not provided', () => {
             expect(
-                getWrapper({ isShown: undefined })
+                getWrapper({} as RadarAnimationProps)
                     .find('.radar')
                     .exists(),
             ).toBe(true);
         });
 
         test('should be shown when isShown is true', () => {
             expect(
-                getWrapper({ isShown: true })
+                getWrapper({ isShown: true } as RadarAnimationProps)
                     .find('.radar')
                     .exists(),
             ).toBe(true);
         });
 
         test('should not be shown when isShown is false', () => {
             expect(
-                getWrapper({ isShown: false })
+                getWrapper({ isShown: false } as RadarAnimationProps)
                     .find('.radar')
                     .exists(),
             ).toBe(false);
         });
     });
 
     describe('position instance method', () => {
-        test.each([true, false])(`should only position the tether when shown`, isShown => {
+        test.each([true, false])('should only position the tether when shown', isShown => {
             const positionTetherMock = jest.fn();
 
-            const wrapper = getWrapper({ isShown });
+            const wrapper = getWrapper({ isShown } as RadarAnimationProps);
```

Comment:
eslint prettier/prettier rule has been randomly complaining about use of "as" at random (but not all) places. This is one of the things that needs to be figured out why. Could just be a bug in the rule and we may need to disable it.